### DISCUSSION TITLE
Making Towns and Nations Sellable

### DISCUSSION THREAD

**Discussion by TheBeeCollector** [TownyAdvanced/Towny#4552] [Ideas]
While we are used to the idea of selling town blocks, it would be nice to have a way to sell towns and nations. Currently, the only way to do this would be to initiate a behind-the-door agreement to have another player take ownership of a town or nation. The problem with this is you can easily get scammed by the other player making the informal transaction. So I'm proposing an official way to do this via towny. 

## Commands:

- `n fs [amount]`
- `t fs [amount]`
- `n nfs`
- `t nfs`

This would work just like the plot commands where if no amount is specified the town/nation is sold for free.

## Merge Process:

- All town blocks would be transferred to the buying town.
- All players would keep their town ranks, and the previous mayor will become an assistant to the new town.

Whaddya think?

**Comment by LlmDl** [comment]
Something to consider once we have the current database changes made, in the mean time there is a plugin out there (might be called Towny Market) which allows the selling of Towns.

**Reply by TheBeeCollector** [reply]
Yeah, I've heard of that, unfortunately, it seems to be poorly optimized and causes massive lag when a transaction occurs I'm guessing because the transfer code the dev is using isn't up to par.
